debian/bionic/foreman/control

```diff
@@ -17,7 +17,7 @@ Priority: extra
 Depends: ruby, ruby2.5, bundler (>= 1.3), zlib1g-dev,
          ruby2.5-dev, build-essential,
          rake (>=0.8.4), ${misc:Depends}
-Recommends: foreman-proxy, foreman-debug (= ${binary:Version}), python-minimal
+Recommends: foreman-proxy, foreman-debug (= ${binary:Version}), python-minimal, foreman-service
```

Comment:
You can add the exact version like `foreman-debug`

Reply:
Actually, I don't see a point in recommending the exact same version. It'll work with most other versions too.

Reply:
Fair point. In that case, the dependency should also be relaxed to just foreman?

Reply:
Yeah, I think that's right.